src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2935,4 +2940,7 @@
   <data name="SortBy" xml:space="preserve">
     <value>Sort by</value>
   </data>
+  <data name="RunAsAdmin" xml:space="preserve">
+    <value>Run as administrator</value>
```

Comment:
Isn't there already a string for this being used in the compatibility page?

Reply:
Yes, I think we should change that one because RunAsAdmin is shorter.